packages/svmwrap_nonlopt/svmwrap_nonlopt.4.0.0/opam

```diff
@@ -0,0 +1,74 @@
+opam-version: "2.0"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/svmwrap"
+bug-reports: "https://github.com/UnixJunkie/svmwrap/issues"
+dev-repo: "git+https://github.com/UnixJunkie/svmwrap.git"
+license: "BSD-3-Clause"
+build: ["dune" "build" "-p" name "-j" jobs]
```

Comment:
```suggestion
build: ["dune" "build" "-p" "svmwrap" "-j" jobs]
```
This should fix the build, but it would be saner to make nlwrap a depopt or to separate from `svmwrap` a `svmwrap-nlopt` subpackage

Reply:
maybe I should just rename the opam file in the sources

Reply:
good idea